en/release-notes.md

```diff
@@ -4,7 +4,7 @@ table_of_contents: True
 # Release Notes 2.3
 
 MAAS 2.3 is currently under development. The current release is [MAAS 2.3.0
-beta3][currentrelease]. See [Historical release
+RC2][currentrelease]. See [Historical release
```

Comment:
I really hate when we break in the middle of these links. just saying...

Reply:
noted, thanks.